ckanext/iati_generator/actions/iati.py

```diff
@@ -49,8 +71,9 @@ def get_validated_csv_data(context, resource_id):
                 logs.append(f"Row limit reached ({ROWS_LIMIT}); stopping")
                 break
             try:
-                activity = row_to_iati_activity(row)
-                activities.append(activity)
+                if file_type_enum == IATIFileTypes.ORGANIZATION_MAIN_FILE:
```

Comment:
It's not clear what this function does
For the main **org** file, it return a list of rows but the variable it's called `activities`
Activities and Organization are the two type of IATI files.
We are using this function from `generate_iati_xml` which is really old and probably will be redefined.

We don't actually require this function. We can delete it (if not, let me know)
Also, the `generate_iati_xml` action is only in use in one place, in a blueprint that generates IATI XML based on package, not namespace like we want.

So we need to start step by step.
We are in a stage of getting lists of files and resources related to IATI and investigate how to validate
We will probably need to validate with the internal IATI library, probably not here.